packages/core/js-client/src/clientPeer/__test__/client.spec.ts

```diff
@@ -206,11 +243,15 @@ describe("FluenceClient usage test suite", () => {
           },
         });
 
-        peer.internals.initiateParticle(particle, (stage) => {
-          if (stage.stage === "sendingError") {
-            reject(stage.errorMessage);
-          }
-        });
+        peer.internals.initiateParticle(
+          particle,
+          () => {},
+          (error: Error) => {
+            if (error instanceof SendError) {
```

Comment:
and if it's a different error, test will hang?

Reply:
Good catch